CMakeLists.txt

```diff
@@ -113,6 +113,7 @@ include/OptimizableTypes.h
 include/MLPnPsolver.h
 include/TwoViewReconstruction.h
 include/Config.h
+include/serialize_tuple.h
```

Comment:
why not directly use boost library? It works with me.

Reply:
When I serialized large maps, I had a bug when recovering the MapPoints's observations from the backup. Serializing mObservations directly solved the problem. However those observations contain a tuples. If I'm not mistaken, by default, there is no tuple serialization in Boost. That's why this is required.

Reply:
I didn't meet this kind of problem. Will your program break down with red error? Recover from backup is ok from my side.

Reply:
not sure if this is helpful. https://github.com/UZ-SLAMLab/ORB_SLAM3/pull/310/files#r627844802
But I think add serialize_tuple.h is also a good option:)